### DISCUSSION TITLE
Both ethersv5 and wagmi support

### DISCUSSION THREAD

**Discussion by hiteshjagwani102** [WalletConnect/walletconnect-monorepo#4111] [Developer Support] [answered]
Is it possible to have both ethersv5 and wagmi support, so that I can use both wagmi and ethers libraries??

**Comment by glitch-txs** [comment]
You can use ethers adapter from Wagmi
https://wagmi.sh/react/guides/ethers

**Reply by hiteshjagwani102** [reply]
Hey can yout suggest what could be the best wagmi version to use this code and the web3modal/wagmi sdk together, as web3modal suggests wagmi @1.4.3 but useConnectorClient hook is not available in that version, so I am facing compatibality issues.

**Reply by glitch-txs** [reply, marked as answer]
Try this https://1.x.wagmi.sh/react/ethers-adapters
We'll add support to wagmi v2 later this week
